umbraco/Umbraco-CMS
# V9 Remove System.Configuration

System.Configuration is used for XML based config which we are no longer using Umbraco V9

## Test Notes
* Add project reference to TestData into Web.UI project
* Enable it in appsettings.json
* Visit /LoadTest (Not 100% sure we still use this)
* Ensure it works as it did before
* Alternative way is to visit `/umbraco/surface/umbracotestdata/CreateTree?count=100&depth=5`

```json
"Umbraco": {
    "CMS": {
        "TestData": {
            "Enabled" : true,
        }
    }
}
```
